Dockerfile-native

```diff
@@ -0,0 +1,46 @@
+ARG PRESTO_VERSION
+
+FROM prestocpp/prestocpp-avx-centos:kpai-20221018 as Builder
+
+WORKDIR  /app
+COPY . .
+RUN cd presto-native-execution && \
+    make velox-submodule && \
+    source /opt/rh/gcc-toolset-9/enable && \
+    source velox/scripts/setup-helper-functions.sh && \
+    (mkdir -p third_party && cd third_party && github_checkout aws/aws-sdk-cpp 1.9.96 --depth 1 --recurse-submodules --config advice.detachedHead=false && \
+    cmake_install -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS:BOOL=OFF -DMINIMIZE_SIZE:BOOL=ON -DENABLE_TESTING:BOOL=OFF -DBUILD_ONLY:STRING="s3;identity-management") && \
+    TREAT_WARNINGS_AS_ERRORS=0 make release NUM_THREADS=8 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8 && \
+    mkdir -p prestissimo/lib prestissimo/lib64 && \
+    cp _build/release/presto_cpp/main/presto_server prestissimo && \
```

Comment:
@wanglinsong we are missing copying libraries under /usr/local/lib and /usr/local/lib64 to prestissimo/lib and prestissimo/lib64. Would you please add those.